plugin/pom.xml

```
@@ -25,7 +25,7 @@
     <changelist>-SNAPSHOT</changelist>
     <module.name>${project.groupId}.warnings.ng</module.name>
 
-    <analysis-model-api.version>11.0.1</analysis-model-api.version>
+    <analysis-model-api.version>11.1.0-rc684.c3c88b_b_ca_fd7</analysis-model-api.version>
     <analysis-model-tests.version>${analysis-model-api.version}</analysis-model-tests.version>
```

This should stay the same:
```suggestion
    <analysis-model-tests.version>11.0.1</analysis-model-tests.version>
```